service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -285,15 +293,29 @@ private void sendNotification(String templateId, String email, Map<String, Objec
         }
     }
 
-    private void sendOrderNotificationForLocalAuthority(final CaseDetails caseDetails, final String localAuthorityCode,
-                                                        final String mostRecentUploadedDocumentUrl) {
+    private void sendGeneratedOrderNotificationToLocalAuthority(final CaseDetails caseDetails,
+                                                                final String localAuthorityCode,
+                                                                final String mostRecentUploadedDocumentUrl) {
         Map<String, Object> localAuthorityParameters =
-            orderEmailContentProvider.buildOrderNotificationParametersForLocalAuthority(
+            generatedOrderEmailContentProvider.buildOrderNotificationParametersForLocalAuthority(
                 caseDetails, localAuthorityCode, mostRecentUploadedDocumentUrl);
 
         String recipientEmail = inboxLookupService.getNotificationRecipientEmail(caseDetails, localAuthorityCode);
 
-        sendNotification(ORDER_NOTIFICATION_TEMPLATE, recipientEmail, localAuthorityParameters,
+        sendNotification(ORDER_NOTIFICATION_TEMPLATE_FOR_LA, recipientEmail, localAuthorityParameters,
+            Long.toString(caseDetails.getId()));
+    }
+
+    private void sendOrderIssuedNotificationToHmctsAdmin(final CaseDetails caseDetails,
+                                                         final String localAuthorityCode,
+                                                         final byte[] documentContents) {
+        Map<String, Object> hmctsParameters =
+            orderIssuedEmailContentProvider.buildOrderNotificationParametersForHmctsAdmin(
```

Comment:
what's the difference between this and generatedOrderEmailContentProvider. Should they be separate?

Reply:
yes - generatedOrderEmailContentProvider is only for orders that come from 'Create an order' event. but I want to send a notification from 3 different triggers (CMO, notice of placement and create order event). So I need a new service that can provide to these 3 controllers.
Initially I did actually have them together but then realised they should be separate